docs/UserGuide.md

```diff
@@ -584,5 +584,8 @@ Action | Format, Examples
 * **CLI**: Command Line Interface
 * **GUI**: Graphical User Interface
 * **Inventory**: List of all items stored in StoreMando
-
+* **Item**: An item is made up of an item name, location, quantity, expiry date(optional) and tags(optional). 
+* **Similar Items**: Two items are similar if they have the same spelling for name and location but differ by letter case.
+* **Identical Items**: Two items are identical if they have the same name and location in terms of spelling and letter case, as well as the same expiry date.    
```

Comment:
Items that have the same name and location in terms of spelling and letter case, as well as the same expiry date if it exists for both.

Reply:
Fixed.